Sources/YapDB.swift

```diff
@@ -320,23 +326,23 @@ extension YapDB {
     In this case, the parent is a YapDB.Fetch type. This
     allows for searching of other filters, and even searching inside search results.
     */
-    public class SearchResults: BaseView, YapDatabaseViewProducer, YapDatabaseExtensionRegistrar {
+    open class SearchResults: BaseView, YapDatabaseViewProducer, YapDatabaseExtensionRegistrar {
```

Comment:
I prefer to leave these classes `open` so consumers of this pod can subclass them if that is useful for their application.